presto-spark-base/src/main/java/com/facebook/presto/spark/execution/common/WorkerHiveConnectorProperties.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution.common;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * Represents catalog/<connector>.properties in Presto context. It helps to create a working baseline for Prestissimo worker.
+ */
+public class WorkerHiveConnectorProperties
```

Comment:
Same for this one. Properties like `cache.enabled` will never be used by spark....

Reply:
Yeah that's why it needs to be set to false explicitly as default to avoid unexpected behavior.